extensions/amp-accordion/1.0/accordion.js

```diff
@@ -260,9 +278,14 @@ export function AccordionSection({
 
   useLayoutEffect(() => {
     if (registerSection) {
-      return registerSection(id, defaultExpanded);
+      return registerSection(id, defaultExpanded, onExpand, onCollapse);
     }
-  }, [registerSection, id, defaultExpanded]);
+  }, [registerSection, id, defaultExpanded, onExpand, onCollapse]);
+
+  const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
+  const animate = contextAnimate ?? defaultAnimate;
+  const contentId = `${prefix || 'a'}-content-${id}-${suffix}`;
+  const classes = useStyles();
 
   const expandHandler = useCallback(() => {
     if (toggleExpanded) {
```

Comment:
There's a `else` branch here, where we also want to handle the event. E.g.

```
if (toggleExpanded) {
  // let the toggleExpanded do everything
} else {
  setExpandedState((prev) => {
    const newValue = !prev;
    const {onExpand, onCollapse} = sectionPropsRef.current;
    if (newValue && onExpand) {
      onExpand();
    } else if (!newValue && onCollapse) {
      onCollapse();
    }
    return newValue;
  });
}
```

Reply:
Thank you!  I've updated with the single function for handling both `expand` and `collapse`.